### PR TITLE
Refactor state

### PR DESCRIPTION
**Pull request by aymanhab**
Separate state into viewerState and modelState to keep current model visualization settings separate

### CODE REVIEW

**Review by AlbertoCasasOrtiz — APPROVED** [review]
LGTM. I have reviewed the code and everything seems fine. I have tested locally and in netlify preview.

Only comment is that the README file in the backend is duplicated, we should remove one of them.

File: src/backend/backend/backend/tests.py
Comment (AlbertoCasasOrtiz) [reply src/backend/backend/backend/tests.py:1]: I forgot to mention, I have run the tests on both frontend and backend and everything seems fine except for the tests fixed in #69. After merging it everything should be fine.

**Comment by aymanhab** [comment]
Thanks @AlbertoCasasOrtiz I resolved merge conflict of the tests due to #69 and now need another review.

File: src/backend/README_1.md
Comment (aymanhab) [reply src/backend/README_1.md:1]: A previous merge resulted in wiping out the instructions for backend, this attempts to restore it

**Comment by AlbertoCasasOrtiz** [comment]
I just run all tests in frontend and backend and tested manually. Everything seems to work properly.

**Pull request merged**
